CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ more about how to use the Metrics API.
 - Don't run onCrashedLastSession for nil Events (#3785)
 - Redistributable static libraries should never be built with module debugging enabled (#3800)
 
+### Features
```

Comment:
There is an already existing features header in the current unreleased section. I believe this needs to move up a couple of lines.